tracing-core/src/span.rs

```diff
@@ -60,7 +60,13 @@ enum CurrentInner {
 impl Id {
     /// Constructs a new span ID from the given `u64`.
     ///
-    /// **Note**: Span IDs must be greater than zero.
+    /// <div class="information">
+    ///     <div class="tooltip ignore" style="">ⓘ<span class="tooltiptext">Note</span></div>
+    /// </div>
+    /// <div class="example-wrap" style="display:inline-block">
+    /// <div class="example-wrap" style="display:inline-block">
+    /// <pre class="ignore" style="white-space:normal;font:inherit;">
+    /// <strong>Note</strong>: Span IDs must be greater than zero.</pre></div>
```

Comment:
formatting effects need to be scoped here as well

![Screenshot from 2020-06-29 12-04-17](https://user-images.githubusercontent.com/1993852/86045613-b0f78800-ba00-11ea-869f-2b496ffb3758.png)